cmd/fluxsvc/main.go

```diff
@@ -181,11 +184,25 @@ func main() {
 	}
 
 	// Release workers.
+	waitForWorkers := make(chan struct{})
+	shutdownWorkers := make(chan struct{})
 	{
-		worker := release.NewWorker(rjs, instancer, releaseMetrics, logger)
-		releaseTicker := time.NewTicker(time.Second)
-		defer releaseTicker.Stop()
-		go worker.Work(releaseTicker.C)
+		workers := sync.WaitGroup{}
+
+		// for each worker goroutine
+		{
+			workers.Add(1)
+			worker := release.NewWorker(rjs, instancer, releaseMetrics, logger)
+			go func() {
+				defer workers.Done()
+				worker.Work(shutdownWorkers)
+			}()
```

Comment:
Mmm yes, that would keep main.go tidier, which would be pleasing. I was trying not to intervene too much, leaving larger changes for the larger overhaul (probably that which addresses #256).